src/renderer/components/screens/AccountDeletionScreen/styles.module.scss

```diff
@@ -0,0 +1,60 @@
+.AccountDeletionScreen {
+  flex-grow: 1;
+  .Background {
+    background-size: cover;
+    // TODO find a better fire/lava image
```

Comment:
Let's find the right picture as part of this PR, otherwise it'll stay forever hehe

Reply:
I mentioned it on the todo issue, I asked some people, I don't have an answer yet, if you also have fire/lava/campfire/danger pictures that you took and have the rights to you could suggest them in the desktop group.
This does not need to block this first pr.

Reply:
I'd remove this TODO then and just leave it as is. If there's no answer, it will be merged and likely stay forever until someone else makes another PR, but I'd not consider this a TODO anymore